fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerMultiTargetTest.java

```diff
@@ -256,8 +265,9 @@ void testMultiTargetMismatchStateFailure() {
         assertAllValidated(indexes);
     }
 
-    @Test
-    void testMultiTargetPartlyBuildFailure() {
+    @ParameterizedTest
+    @ValueSource(ints = {0, 1, 2, 3, 4, 5, 6, 7})
+    void testMultiTargetPartlyBuildFailure(int reverseSeed) {
```

Comment:
This test should call `assertAllValidated`

Reply:
This test asserts typeStamp failures. But I check that `assertAllValidated` is added to the tests that successfully finish indexing.